### PR TITLE
meta: Update readme

### PR DESCRIPTION
**Pull request by jp7677**

### CODE REVIEW

**Comment by jp7677** [comment]
Hi @Faalagorn, may I ask you to have a short look over my planned readme updates. I would love to have a second look to ensure that what I wrote makes any sense ;).

File: README.md [hunk 1]
```diff
@@ -65,6 +82,7 @@ The actual unit tests can be run with `nvapi64-tests.exe [@unit-tests]` to valid
 
 - [DXVK](https://github.com/doitsujin/dxvk)
 - [DXVK-AGS](https://github.com/doitsujin/dxvk-ags)
+- [VKD3D-Proton](https://github.com/HansKristian-Work/vkd3d-proton)
 - [NVAPI](https://docs.nvidia.com/gameworks/content/gameworkslibrary/coresdk/nvapi/group__dx.html)
 - [NVAPI Open Source SDK](https://download.nvidia.com/XFree86/nvapi-open-source-sdk/)
 - [https://github.com/SveSop/nvapi_standalone](https://github.com/SveSop/nvapi_standalone)
```
Comment (Faalagorn) [inline README.md:88]: Shouldn't that be changed to a text link?

File: README.md [hunk 2]
```diff
@@ -37,11 +34,31 @@ Pre-built binaries of release versions are available at [https://github.com/jp76
```
Comment (Faalagorn) [inline README.md:34]: Prebuilt: https://en.wiktionary.org/wiki/prebuilt

**Review by Faalagorn — CHANGES_REQUESTED** [review]
I suggested for changing _entrypoints_ to [_entry points_](https://en.wikipedia.org/wiki/Entry_point) as I guess it's the correct wording, other than that it's basically just nitpicking. There might be some extra or missing commas here and there, but I didn't catch anything glaring.

Other than that, it looks clear and hopefully easy to understand, so it all sound fine! :+1:

File: README.md [hunk 3]
```diff
@@ -1,30 +1,27 @@
...
 
-It also implements several methods for adapter/display topology and system information. Some of those are a requirement for enabling other NVIDIA related technologies.
-
-This implementation has been mostly tested with Unreal Engine 4, with the game `Assetto Corsa Competizione` and several UE4 technology demos. Unreal Engine 4 utilizes `SetDepthBoundsTest`, newer engine versions also utilize `BeginUAVOverlap`/`EndUAVOverlap`. Forwarding those methods may yield up to 4% extra performance, although this heavily depends on setup and scenery.
-
-Basic topology and system information (vendor ID, driver version and others) has been mostly tested with `GPU Caps Viewer`. The game `Get Even`, which verifies the driver version during launch, starts fine with this implementation.
+While originally being developed for usage with Unreal Engine 4, most notably for `Assetto Corsa Competizione`, more entrypoint have been added for enabling other NVIDIA related technologies. DXVK-NVAPI does not and will never cover the complete surface of NVIDIA's NVAPI. It is not meant as a full replacement, but rather as an addition to DXVK and VKD3D-Proton to enable several GPU features.
```
[the remark this answers is not in the document]
Comment (jp7677) [reply README.md:14]: Shouldn't that be "entry points have been added" (plural)?

[the remark this answers is not in the document]
Comment (Faalagorn) [reply README.md:14]: Yeah, my bad, good catch :P

**Comment by jp7677** [comment]
@Faalagorn Thanks a lot for the super fast review! Very cool!
All changes should be in. I'm awaiting some feedback for the Lutris part and then the readme changes are ready to merge.

**Pull request merged**